### ISSUE TITLE
Zoe PreProcessor errors

### ISSUE THREAD

**Issue by dkamajix** [Fannovel16/comfyui_controlnet_aux#81]
Hello,
Im getting an error with the Zoe Preprocessor.
I have tried to uninstall and reinstall.. but no go.
All other Depth Preprocessors working except zoe.
Leres and Midas ok - Zoe I get an error. Below:
Error occurred when executing Zoe-DepthMapPreprocessor:
![image](https://github.com/Fannovel16/comfyui_controlnet_aux/assets/143731847/73b4907a-d275-4bc8-896a-9d0518ca40f2)

expected size to be one of int or Tuple[int] or Tuple[int, int] or Tuple[int, int, int], but got size with types [, ]

File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 153, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 83, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 76, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\node_wrappers\zoe.py", line 18, in execute
out = common_annotator_call(model, image, resolution=resolution)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\utils.py", line 46, in common_annotator_call
np_result = model(np_image, output_type="np", detect_resolution=detect_resolution, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\__init__.py", line 49, in __call__
depth = self.model.infer(image_depth)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\depth_model.py", line 126, in infer
return self.infer_with_flip_aug(x, pad_input=pad_input, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\depth_model.py", line 110, in infer_with_flip_aug
out = self._infer_with_pad_aug(x, pad_input=pad_input, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\depth_model.py", line 88, in _infer_with_pad_aug
out = self._infer(x)
^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\depth_model.py", line 55, in _infer
return self(x)['metric_depth']
^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1518, in _wrapped_call_impl
return self._call_impl(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1527, in _call_impl
return forward_call(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\zoedepth\zoedepth_v1.py", line 144, in forward
rel_depth, out = self.core(x, denorm=denorm, return_rel_depth=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1518, in _wrapped_call_impl
return self._call_impl(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1527, in _call_impl
return forward_call(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\base_models\midas.py", line 265, in forward
x = self.prep(x)
^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\base_models\midas.py", line 189, in __call__
return self.normalization(self.resizer(x))
^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\src\controlnet_aux\zoe\zoedepth\models\base_models\midas.py", line 176, in __call__
return nn.functional.interpolate(x, (height, width), mode='bilinear', align_corners=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "G:\WORKSPACE AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\nn\functional.py", line 3924, in interpolate
raise TypeError(

**Comment by Fannovel16** [comment]
Fixed at https://github.com/Fannovel16/comfyui_controlnet_aux/commit/d67fe4c44e5b035ed68ed0d481b8b0ebf4bad41f

**Comment by dkamajix** [comment]
hi - sorry how do I apply the fix?
I dont understand your comment above
thanks

**Comment by dkamajix** [comment]
never mind - went in edited the code and it worked like a charm,.
thanks
